src/app/objects/resume.ts

```diff
@@ -0,0 +1,57 @@
+import { template } from "./template";
+
+export interface resume {
+  template: template,
+  header: header,
+  workExps: workExp[],
+  educations: education[],
+  certificates?: string[],
+  skills: string[],
+  summary: string
+}
+
+export interface header {
+  id: number;
+  firstName: string;
+  lastName: string,
+  profession: string,
+  email: string,
+  phone: string,
+  address: {
+    city: string,
+    state: string,
+    zipCode: string
+  }
+}
+
+export interface workExp {
+  title: string,
+  company: string,
+  address: {
+    country: string,
+    state: string,
+    city: string,
+  }
+  currentWork: boolean,
```

Comment:
Can you make it so that if there's no end month and no end year, it's by default the current work? This way we can get rid of this boolean value, evaluate end month and end year and set things to be for example 2022-Now

Reply:
Fixed